packages/@aws-cdk/aws-elasticloadbalancing/lib/load-balancer.ts

```diff
@@ -141,10 +141,20 @@ export interface HealthCheck {
   readonly timeout?: Duration;
 }
 
+/**
+ * An object that has instance object.
+ */
```

Comment:
Should be explaining the motivation behind having an `IInstance` class also. Give another look at all your doc strings and see if they can be improved. For example,

```ts
/**
 * Ec2 instance
 */
readonly ec2Instance?: Instance;
```

The docstring isn't really giving us more information than the property name.